From 905e9871acfb5e9f8424ebb09c91df4df49fb4b6 Mon Sep 17 00:00:00 2001
From: Oleksandr Natalenko <oleksandr@natalenko.name>
Date: Wed, 4 Sep 2024 18:27:56 +0200
Subject: [PATCH 39/41] Revert "cpufreq/amd-pstate: make amd_pstate_msr
 non-static"

This reverts commit cc39a4b11ed8ab87266756dec85ac075bcccae9d.
---
 arch/x86/kernel/acpi/cppc.c  | 1 -
 drivers/cpufreq/amd-pstate.c | 2 +-
 2 files changed, 1 insertion(+), 2 deletions(-)

diff --git a/arch/x86/kernel/acpi/cppc.c b/arch/x86/kernel/acpi/cppc.c
index 8551cc127..c0c77e5f2 100644
--- a/arch/x86/kernel/acpi/cppc.c
+++ b/arch/x86/kernel/acpi/cppc.c
@@ -19,7 +19,6 @@ enum amd_pref_core {
 };
 static enum amd_pref_core amd_pref_core_detected;
 static u64 boost_numerator;
-extern bool amd_pstate_msr;
 
 /* Refer to drivers/acpi/cppc_acpi.c for the description of functions */
 
diff --git a/drivers/cpufreq/amd-pstate.c b/drivers/cpufreq/amd-pstate.c
index e206596f0..5bd06c40f 100644
--- a/drivers/cpufreq/amd-pstate.c
+++ b/drivers/cpufreq/amd-pstate.c
@@ -86,7 +86,7 @@ static struct cpufreq_driver amd_pstate_epp_driver;
 static int cppc_state = AMD_PSTATE_UNDEFINED;
 static bool cppc_enabled;
 static bool amd_pstate_prefcore = true;
-bool amd_pstate_msr = true;
+static bool amd_pstate_msr = true;
 static struct quirk_entry *quirks;
 
 /*
-- 
2.45.2.606.g9005149a4a

